CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+1.0.0
+========
+- !!! (maybe breaking change) add tzinfo=UTC for returned datetime field
```

Comment:
Im ok on adding a breaking change since we are just going for a new major release, but then lets clarify it is it or not.

Reply:
I think this is not a breaking change for 99.9% of use cases

great, I'll merge this request now and make a new one a little later and after that we'll make a release